app/economy/eth.py

```diff
@@ -24,7 +24,7 @@
 def get_network_details(network):
     bounty_contract_address = None
     if network == 'mainnet':
-        bounty_contract_address = '0x4e1799e6e54d848394b0b90593e789e939a9f801'
+        bounty_contract_address = '0xb10700b5ece20a3c65b047f76fd3dc13720bd30e'
```

Comment:
Should this be the same address as above?  In the `abi.js` file the mainnet address used is `0x066128b9f7557b5398db3d4ed141f2e64245ffa1`.

Reply:
i... think so.  i may have just gotten this wrong (or i wrote this PR when there was a diff version of `StandardBoutnties` deployed to the mainnet.